api/lib/validation/attributeValidator.js

```diff
@@ -76,36 +83,81 @@ var validateRecurringScheduleAttributeValue = function(inputRecurringSchedules)
     if(inputRecurringSchedules[i].instance_min_count >= 
       inputRecurringSchedules[i].instance_max_count) {
       let error = createErrorResponse('recurring_schedule.instance_min_count',
-      'recurring_schedule.instance_min_count and recurring_schedule.instance_max_count ' + 
-      'values are not compatible',inputRecurringSchedules[i],'instance_min_count ' + 
-       inputRecurringSchedules[i].instance_min_count + ' is higher or equal to ' + 
-       'instance_max_count ' + inputRecurringSchedules[i].instance_max_count +
-       ' in recurring_schedule :[' + i + ']');
+          'recurring_schedule.instance_min_count and recurring_schedule.instance_max_count ' + 
+          'values are not compatible',inputRecurringSchedules[i],'instance_min_count ' + 
+          inputRecurringSchedules[i].instance_min_count + ' is higher or equal to ' + 
+          'instance_max_count ' + inputRecurringSchedules[i].instance_max_count +
+          ' in recurring_schedule :[' + i + ']');
       attributeValueErrors[errorCount++] = error;
     }
     if(!compareTimes(inputRecurringSchedules[i].start_time,
```

Comment:
@fraenkel We don't allow days_of_week and days_of_month as part of the same schedule. The PUT will fail in that case since the schema validation only allows for one of them in a schedule. However, we allow start_date, end_date to be specified with either of the other two.

Yes, It will be valid. It first checks the time and then the dates. So if the end_time > start_time and the dates are the same, the check will pass. So, shouldn't be a problem.